### ISSUE TITLE
Implement Multi-Object delete?

### ISSUE THREAD

**Issue by guilleart** [qoobaa/s3#59]
Are there any plans to implement this feature?

**Comment by haanhduclinh** [comment]
@qoobaa hi. Is that features available now?

**Comment by qoobaa** [comment]
@haanhduclinh no, it's not implemented. The repository isn't actively maintained anymore.
